### DISCUSSION TITLE
Compilation and publishing

### DISCUSSION THREAD

**Discussion by Yveske1966** [AvaloniaUI/Avalonia#17849] [Q&A] [answered]
Visual studio 17.13.0 Preview 2.1 with Avalonia 11.2.3

During compilation on Android the compiler searches SelectPicturesAvalonia.Android.dll in obj\Debug\net9.0-android35.0\Avalonia\Avalonia, which is 1 level too deep. Creating this extra Avalonia directory and copying the content is sufficient to finish compilation.

Publishing on Windows desktop only succeeds for x86, but bombs out on x64, arm and arm64
Assets file '...Desktop\obj\wappublish\win-x64\project.assets.json' not found. Run a NuGet package restore to generate this file.

Can anyone confirm this or know a workaround ?

**Comment by Yveske1966** [comment]
On Mac M1 with Visual Studio Code v1.96 both IOS and MacOS

Error NETSDK1005: Assets file '...obj/project.assets.json' doesn't have a target for 'net9.0'. Ensure that restore has run and that you have included 'net9.0' in the TargetFrameworks for your project.

Same question. Can anyone confirm this or know a workaround ?

**Reply by IsaqueS** [reply]
Nop, still finding an solution...

**Reply by Yveske1966** [reply]
Thanks for confirming !

**Comment by Yveske1966** [comment, marked as answer]
Situation solved itself after updates
